homeassistant/components/notify/smtp.py

```diff
@@ -47,7 +47,7 @@
     vol.Optional(CONF_SERVER, default=DEFAULT_HOST): cv.string,
     vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
     vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int,
-    vol.Optional(CONF_STARTTLS, default=DEFAULT_STARTTLS): cv.boolean,
+    vol.Optional(CONF_ENCRYPTION, default=DEFAULT_ENCRYPTION): cv.string,
```

Comment:
We should limit the options to valid entries. See our [validation docs](https://home-assistant.io/developers/development_validation/#limit-the-values).